Documentation/deploy-master.md

```diff
@@ -405,7 +405,7 @@ A successful response should look something like:
 {
   "major": "1",
   "minor": "0",
-  "gitVersion": "v1.0.6",
+  "gitVersion": "v1.0.7",
   "gitCommit": "388061f00f0d9e4d641f9ed4971c775e1654579d",
```

Comment:
i honestly don't care about this output being kept up to date. How about we just stop modifying it as of v1.0.6?